Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/BabylonReactNative.vcxproj

```diff
@@ -192,7 +192,7 @@
         NativeXr.lib;
         nvtt.lib;
         OGLCompilerd.lib;
-        openxr_loader.lib;
+        openxr_loaderd.lib;

```

Comment:
Aren't these changes part of @chrisfromwork's PR? Not sure why they are showing up here.

Reply:
These changes were recommended by chris, I added them to this PR so that it would pass windows CI. We can check in #188 and I can rebase, else we pretty much have the same goal here, I'm just integrating my changes from the new latest commit.

Reply:
yeah in moving to the newest master we'll see breaks without this change